prover/vk_setup_data_generator_server_fri/src/vk_generator.rs

```diff
@@ -27,11 +56,125 @@ fn main() -> anyhow::Result<()> {
     generate_basic_circuit_vks().context("generate_basic_circuit_vks()")
 }
 
+fn get_all_basic_circuits(
+    geometry: &GeometryConfig,
+) -> Vec<
+    ZkSyncBaseLayerCircuit<GoldilocksField, VmWitnessOracle<GoldilocksField>, Poseidon2Goldilocks>,
+> {
+    vec![
+        ZkSyncBaseLayerCircuit::MainVM(ZkSyncUniformCircuitInstance {
+            witness: AtomicCell::new(Some(VmCircuitWitness::<
```

Comment:
None is enough, correct and serves as self-protection for VK generation

Reply:
thanks - applied these changes in test_harness directly.